apps/openmw/mwrender/objects.cpp

```diff
@@ -45,7 +45,7 @@ void Objects::resetSettings()
     mOQNSettings.querymargin = Settings::Manager::getFloat("queries margin", "OcclusionQueries");
     mOQNSettings.maxBVHOQLevelCount = Settings::Manager::getInt("max BVH OQ level count", "OcclusionQueries");
     mOQNSettings.securepopdistance = Settings::Manager::getFloat("min pop in distance", "OcclusionQueries");
-    mOQNSettings.OQMask = Mask_OcclusionQuery;
+    mOQNSettings.OQMask = SceneUtil::Mask_OcclusionQuery;
```

Comment:
We are trying to get away from this...  see: 
https://gitlab.com/OpenMW/openmw/-/merge_requests/178